django_loci/apps.py

```diff
@@ -15,11 +25,21 @@ def __setmodels__(self):
         from .models import Location
         self.location_model = Location
 
+    def test_geocoder(self):
+        try:
+            location = geocode('Red Square')
+            assert location is not None
+        except AssertionError:
+            if DJANGO_LOCI_GEOCODE_STRICT_TEST:
+                raise ImproperlyConfigured('Geocode utility is improperly configured')
+            self.logger.exception('Geocode test failed')
+
     def ready(self):
         import leaflet
         leaflet.app_settings['NO_GLOBALS'] = False
         self.__setmodels__()
         self._load_receivers()
+        self.test_geocoder()
```

Comment:
rename this method to `_test_geocoder`, it makes sense to mark it as private and is also consistent with the other methods